src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -164,38 +157,39 @@ public boolean add(DatenFilm aFilm)
     public synchronized void clear() {
         nr = 1;
         neueFilme = false;
+        indexes.clear();
         super.clear();
     }
 
-    public void cleanList() {
-        // für den BR: alle Filme mit Thema "BR" die es auch in einem anderen Thema gibt löschen
-        // wird vorerst nicht verwendet: findet nur ~200 Filme von über 3000
-        int count = 0;
-        final SimpleDateFormat sdfClean = new SimpleDateFormat(DATUM_ZEIT_FORMAT);
-        Log.sysLog("cleanList start: " + sdfClean.format(System.currentTimeMillis()));
-        ListeFilme tmp = this.stream().filter(datenFilm -> datenFilm.arr[DatenFilm.FILM_SENDER].equals(Const.BR))
-                .filter(datenFilm -> datenFilm.arr[DatenFilm.FILM_THEMA].equals(Const.BR))
-                .collect(Collectors.toCollection(ListeFilme::new));
-
-        for (DatenFilm tFilm : tmp) {
-            for (DatenFilm datenFilm : this) {
-                if (datenFilm.arr[DatenFilm.FILM_SENDER].equals(Const.BR)) {
-                    if (!datenFilm.arr[DatenFilm.FILM_THEMA].equals(Const.BR)) {
-                        if (datenFilm.arr[DatenFilm.FILM_URL].equals(tFilm.arr[DatenFilm.FILM_URL])) {
-                            this.remove(tFilm);
-                            ++count;
-                            break;
-                        }
-                    }
-                }
-            }
-        }
-
-        tmp.clear();
-
-        Log.sysLog("cleanList stop: " + sdfClean.format(System.currentTimeMillis()));
-        Log.sysLog("cleanList count: " + count);
-    }
+//    public void cleanList() {
+//        // für den BR: alle Filme mit Thema "BR" die es auch in einem anderen Thema gibt löschen
+//        // wird vorerst nicht verwendet: findet nur ~200 Filme von über 3000
+//        int count = 0;
+//        final SimpleDateFormat sdfClean = new SimpleDateFormat(DATUM_ZEIT_FORMAT);
+//        Log.sysLog("cleanList start: " + sdfClean.format(System.currentTimeMillis()));
+//        ListeFilme tmp = this.stream().filter(datenFilm -> datenFilm.arr[DatenFilm.FILM_SENDER].equals(Const.BR))
+//                .filter(datenFilm -> datenFilm.arr[DatenFilm.FILM_THEMA].equals(Const.BR))
+//                .collect(Collectors.toCollection(ListeFilme::new));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)